content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -130,9 +130,12 @@ directories:
 
 
 ## Kubeflow installation
-Run the following commands to set up and deploy Kubeflow.
 
-1. Download the kfctl {{% kf-latest-version %}} release from the
+**Note**: kfctl is currently available for Linux and macOS users only. If you use Windows, you can install kfctl on Windows Subsystem for Linux (WSL). Refer to the official [instructions](https://docs.microsoft.com/en-us/windows/wsl/install-win10) for setting up WSL.
+
+Run the following commands to set up and deploy Kubeflow:
+
+1. Download the latest kfctl {{% kf-latest-version %}} release from the
   [Kubeflow releases 
   page](https://github.com/kubeflow/kfctl/releases/tag/{{% kf-latest-version %}}).
 
```

Comment:
Can you also do some minor linting if you have time? This can help with readability, future reviews, and improved UX for KF users.

---

For example, can you please add blank lines between code blocks and normal text paragraphs below? Also, for better Markdown formatting, add `shell` or `bash` to command-line code blocks. 

Lines 139 and 144:

```diff
- 1. Unpack the tar ball
+ 1. Extract the archived TAR file:
+ 
-       ```
+       ```shell
      tar -xvf kfctl_{{% kf-latest-version %}}_<platform>.tar.gz		      tar -xvf kfctl_{{% kf-latest-version %}}_<platform>.tar.gz
+ 
      ```
```

(repeat for line 44, if you can, please)

---

Line 151: please add a blank line as follows:

```diff
### Single user
+       
Run the following commands to set up and deploy Kubeflow for a single user without any authentication.
```

same between the lines 204-205, 291-292, 302-303, 308-309

---

Lines 285-287: Can you please split the complex sentence into simpler ones for readability/better UX?


```diff
    **Note**: If any of the parameters changed after the initial Kubeflow deployment, you
-     will need to manually update these parameters in the secret `appid-application-configuration`
+     will need to manually update these parameters in the secret `appid-application-configuration`.
-     then restart authservice by running the command `kubectl rollout restart sts authservice -n istio-system`.
+     Then, restart authservice by running the command `kubectl rollout restart sts authservice -n istio-system`.
```

---

Line 298: Please change the order of words and fix the grammar:

```diff
## Next steps
+       
- Please follow the steps in [Exposing the Kubeflow dashboard with DNS and TLS termination]
- (../authentication/#exposing-the-kubeflow-dashboard-with-dns-and-tls-termination) to secure the Kubeflow 
- dashboard with HTTPS, then you will have the required DNS name as Kubeflow FQDN to enable the OIDC flow for 
- AppID:		
+ To secure the Kubeflow dashboard with HTTPS, follow the steps in 
+ [Exposing the Kubeflow dashboard with DNS and TLS termination](../authentication/#exposing-the-kubeflow-dashboard-with-dns-and-tls-termination).
+ Then, you will have the required DNS name as Kubeflow FQDN to enable the OIDC flow for AppID:

---

Line 308: Please change to capital "R" in "restart" (use sentence case, basically):

```diff
- 3. restart the pod `authservice-0`:		
+ 3. Restart the pod `authservice-0`:		
```

---

Please rephase line 313:

```diff
- Then visit `https://<kubeflow-FQDN>/`, it should redirect you to AppID for authentication.
+ Then, visit `https://<kubeflow-FQDN>/`. The page should redirect you to AppID for authentication.

(or "site" or whatever you think is best.)
```

---

Thank you!